### PR TITLE
openstack-staging: node openstack-rpm-packaging

### PR DESCRIPTION
**Pull request by JanZerebecki**
usually has an executor free, use that instead. The job
openstack-staging only calls other jobs but still needs one slot.
cloud-cleanvm only has one executor slot.

### CODE REVIEW

File: jenkins/ci.opensuse.org/openstack-staging.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@
           type: slave
           name: slave
           values:
-            - cloud-cleanvm
+            - openstack-rpm-packaging
```
Comment (dirkmueller) [inline jenkins/ci.opensuse.org/openstack-staging.yaml]: how is that related to rpm-packaging? you could use openstack-trackupstream

Comment (JanZerebecki) [reply jenkins/ci.opensuse.org/openstack-staging.yaml]: If it had one more slot, I would. Using the only slot from it would be bad.

Comment (JanZerebecki) [reply jenkins/ci.opensuse.org/openstack-staging.yaml]: The openstack-trackupstream node has the label openstack, that would also be more fitting.

Comment (dirkmueller) [reply jenkins/ci.opensuse.org/openstack-staging.yaml]: I can add a 2nd slot, but isn't having one slot a feature here? we don't want source commits and staging to happen at the same time..

Comment (JanZerebecki) [reply jenkins/ci.opensuse.org/openstack-staging.yaml]: As this job was split from the cleanvm job, it needs a another slot. We don't get parallel execution because of       sequential:true and block:true below. There is no `cloud-trigger` in ci.opensuse.org.

Comment (JanZerebecki) [reply jenkins/ci.opensuse.org/openstack-staging.yaml]: Changed to: openstack which currentyl only ends up on openstack-trackupstream-sle12.

**Comment by dirkmueller** [comment]
Ok. At some point I would really like to understand why we're adding this to ci.o.o..

**Comment by JanZerebecki** [comment]
Adding what? Nothing in this patch or the one that it fixes adds anything to ci.o.o, they only modify the exiting staging jobs to use osc release.
